File: Old Java Code/Client/org/hardgate/familytree/ParishBatch.java
/*
 * Created on 19-May-2005
 *
 */
package org.hardgate.familytree;

public class ParishBatch {
    
    private String batch;
    private String parishID;
    private String parish;
    private String startYear;
    private String endYear;
    private String comments;
    
    public ParishBatch(String batch, String parishID) {
        this.batch = batch;
        this.parish = "";
        this.parishID = parishID;
        this.startYear = "";
        this.endYear = "";
        this.comments = "";
    }

    public String getBatch() {
        return batch;
    }

    public String getParish() {
        return parish;
    }

    public String getParishID() {
        return parishID;
    }

    public String getComments() {
        return comments == null ? "" : "(" + comments + ")";
    }

    public void setComments(String comments) {
        this.comments = comments;
    }

    public String getEndYear() {
        return endYear;
    }

    public void setEndYear(String endYear) {
        this.endYear = endYear;
    }

    public String getStartYear() {
        return startYear;
    }

    public void setStartYear(String startYear) {
        this.startYear = startYear;
    }

	public void setParish(String parish) {
		this.parish = parish;
	}

}

File: FTAnalyser/Core/ParishBatch.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FTAnalyzer
{
    public class ParishBatch
    {

        public string Batch { get; private set; }
        public string ParishID { get; private set; }
        public Parish Parish { get; set; }
        public string StartYear { get; set; }
        public string EndYear { get; set; }
        private string comments;

        public ParishBatch(String batch, String parishID)
        {
            this.Batch = batch;
            this.Parish = null;
            this.ParishID = parishID;
            this.StartYear = "";
            this.EndYear = "";
            this.comments = "";
        }

        #region Properties

        public string Comments
        {
            get { return comments == null ? "" : "(" + comments + ")"; }
            set { this.comments = value; }
        }

        #endregion
    }
}